client/docs/py-primer-dark.html

```diff
@@ -0,0 +1,178 @@
+<html>
+  <head>
+    <style>
+      a {
+        color: #82B1FF;
+      }
+      body {
+        background-color: #333a42;
+        color: #e0e0e0;
+        font-family: Helvetica, Arial;
+        font-size: 14px;
+      }
+      p {
+        width: 616px;
+      }
+      pre {
+        color: white;
+        font-size: 13px;
+        padding-top: 8px;
+        padding-bottom: 8px;
+        width: 632px;
+      }
+      .function-name {
+        color: #e9eded;
+      }
+      .general-statements {
+        color: #c792ea;
+      }
+      .h1 {
+        color: white;
+        font-size: 24px;
+        height: 50px;
+        line-height: 50px;
+        text-align-last: center;
+        width: 616px;
+      }
+      .h2 {
+        background-color: #4242b7;
+        color: #e0e0e0;
+        font-size: 16px;
+        height:32px;
+        line-height: 32px;
+        margin-bottom: 24px;
+        margin-left: -8px;
+        margin-right: -8px;
+        margin-top: 24px;
+        text-align-last: center;
+        width: 640px;
+      }
+      .method-type {
+        color: #decb6b;
+      }
+      .names {
+        color: #6f96e0;
+      }
+      .number {
+        color: #f77669;
+      }
+      .property {
+        color: #669c87;
+      }
+      .single-line-comment {
+        color: #7599a9;
+      }
+      .string {
+        color: #a0b97c;
+      }
+      .transparent {
+        font-size: 0px;
+        height: 0px;
+        opacity: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="h1">
```

Comment:
Why not just use h1, h2 tags (and style those appropriately if needed)? These are (semantically) headers, and one should use the right tags for accessibility reasons.

Reply:
Changed as requested.